policyengine_us/reforms/states/ny/wftc/ny_working_families_tax_credit.py

```diff
@@ -58,7 +58,7 @@ def formula(person, period, parameters):
             is_dependent = person("is_tax_unit_dependent", period)
             age = person("age", period)
             p = parameters(period).gov.irs.dependent.ineligible_age
-            age_eligible = age < p.non_student
+            age_eligible = age <= p.non_student
```

Comment:
Is this a shortcut for changing the IRC's 17 to 18 here in [the bill](https://legislation.nysenate.gov/pdf/bills/2023/S277B)?
>(C) "Qualifying child" or "qualifying children" shall mean as defined 55 in 26 USC §24(c)(1) except that such term shall also include qualifying 56 children who have not attained the age of eighteen.

If so could we instead make it a parameter?

Reply:
Yes, we already have a parameter created - pointed to it instead of the federal age threshold now